sdk/ai/ai-inference-rest/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const SDK_VERSION: string = "1.0.0-beta.2";
```

Comment:
You'll need to add this to the `constantPaths` entry in package.json to allow the azure sdk bot to automatically bump the version here